exercises/en/exc_01_32.py

```diff
@@ -1,16 +1,22 @@
 import pandas as pd
+import altair as alt
+# Load in the csv named "canucks.csv" the same way you did earlier
+# Save it as hockey_players
 
-# Load in the csv named "position_frequencies.csv" you made in exercise 24
-# Save it as position_freq
-
-____ = pd.____('data/____', ____)
+# hockey_players = pd.____('data/____', ____)
 
 # ____
 
-# Use plot.bar() with position_freq
-# Assign a color as Teal and set opacity to 0.5  
+# Use alt.Chart() to create a chart object
+# Use the .mark_bar() to create a bar plot
+# Use the .encode() to specify the column to plot
+# Assign a color as Teal and set opacity to 0.5
 # Don't forget to add a title as "Canuck Player Positions"
 
-# ____ = ____.____.____(____ = ____,
-#                      ____ = ____,
-#                      ____ = ____)
+# position_bar = alt.____(
+#    ____, width=500, height=300).____(
```

Comment:
add the encode function and alt.Chart since this s their first plot ( the next Q you can keep as you have it)